### PR TITLE
Update: Support quick time range select #92 && fix Date picker style

### PR DESCRIPTION
**Pull request by shiluo34**
### update quick picker options
1571557283387_ pic
##### before
image
##### updated
image

### CODE REVIEW

**Comment by wu-sheng** [comment]
@shiluo34 You follow #92 , right?

**Comment by TinyAllen** [comment]
@shiluo34 
Ooops! A little of problem. The time quick select is not suitable for the utc, before you set time you should computer the utc first
 
image

**Comment by shiluo34** [comment]
image
add utc for picker

File: src/components/rk-date.vue [hunk 1]
```diff
@@ -202,8 +203,9 @@ export default {
       this.show = this.$el.contains(e.target) && !this.disabled;
     },
     quickPick(type){
-      const end = new Date();
-      const start = new Date();
+      const utc = window.localStorage.getItem('utc') || 8; //默认东8区
```
Comment (wu-sheng) [inline src/components/rk-date.vue]: Remove Chinese character, this is not allowed in Apache repo. And don't put the default UTC timezone, keep it strictly as same as the page shows.

Comment (shiluo34) [reply src/components/rk-date.vue]: ok

**Comment by wu-sheng** [comment]
@JaredTan95 Update the submodule PR, please.

**Pull request merged**